governance/program/src/state/governance.rs

```diff
@@ -104,7 +104,7 @@ pub struct GovernanceV2 {
     /// Reserved space for versions v2 and onwards
     /// Note 1: V1 accounts must be resized before using this space
     /// Note 2: The reserved space should be used from the end to also allow the config to grow if needed
-    pub reserved_v2: Reserved120,
+    pub reserved_v2: Reserved112,
```

Comment:
There is no need to take this reserved space because we reallocate the account to `max_size` in `set_governance_config`